model/AliasHandler.php

```diff
@@ -441,6 +441,11 @@ public function delete() {
             return false;
         }
 
+        if ($this->can_delete) {
```

Comment:
Oooops...
I've forgotten the `! ` while transferring the patch from my production system to my local git copy...